etc/helm/pachyderm/templates/etcd/statefulset.yaml

```diff
@@ -37,10 +37,10 @@ spec:
       containers:
       - args:
         - '"/usr/local/bin/etcd" "--listen-client-urls=http://0.0.0.0:2379" "--advertise-client-urls=http://0.0.0.0:2379"
-          "--data-dir=/var/data/etcd" "--auto-compaction-retention=1" "--max-txn-ops=10000"
+          "--data-dir=/var/data/etcd" "--auto-compaction-retention=1" "--max-txn-ops={{ .Values.etcd.maxTxnOps }}"          
           "--max-request-bytes=52428800" "--quota-backend-bytes=8589934592" "--listen-peer-urls=http://0.0.0.0:2380"
           "--initial-cluster-token=pach-cluster" "--initial-advertise-peer-urls=http://${ETCD_NAME}.etcd-headless.${NAMESPACE}.svc.cluster.local:2380"
-          "--initial-cluster={{ range $i, $e := until (.Values.etcd.dynamicNodes | int) }}etcd-{{$e}}=http://etcd-{{$e}}.etcd-headless.${NAMESPACE}.svc.cluster.local:2380{{ end }}"'
+          "--initial-cluster={{ range $i, $e := until (.Values.etcd.dynamicNodes | int) }}etcd-{{$e}}=http://etcd-{{$e}}.etcd-headless.${NAMESPACE}.svc.cluster.local:2380,{{ end }}"'        
```

Comment:
nit: any chance you could remove the trailing whitespace?

Reply:
sure, done